src/openms/include/OpenMS/ANALYSIS/ID/FragmentIndex.h

```diff
@@ -39,6 +39,41 @@ namespace OpenMS
       float precursor_mz;  // charge 1 precursor m/z
     };
 
+    struct sPeptide
+    {
+      UInt32 protein_idx_;
+      UInt32 modification_idx;
+      std::pair<Size, Size> sequence;
+      float precursor_mz;
+    };
+
+    /**
+     * @brief Every potential Peptide/Protein has such an struct. Inside the number of peaks-to-Fragment hits are safed
+     */
+    struct SpectrumMatch
+    {
+      uint32_t num_matched_{};       ///< Number of peaks-fragment hits
+      uint32_t precursor_charge_{};  ///< The precursor_charged used for the performed search
```

Comment:
could be uint16_t. if it is listed directly before or after with isotope_error we would safe another word (64bit)